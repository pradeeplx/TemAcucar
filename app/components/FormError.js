import React from 'react'
import Colors from "../Colors"
import TextBox from "./TextBox"

export default FormError = ({ message }) => (
  <TextBox style={{
    color: Colors.red,
    marginTop: 24,
  }}>
    {message}
  </TextBox>
)
