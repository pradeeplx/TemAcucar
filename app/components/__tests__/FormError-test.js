import 'react-native'
import React from 'react-native'
import FormError from '../FormError'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <FormError message="Bug!" />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});