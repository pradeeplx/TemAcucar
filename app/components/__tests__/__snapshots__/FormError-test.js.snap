exports[`test renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#654C50",
        "fontFamily": "Avenir",
        "fontSize": 18,
      },
      Array [
        Object {
          "marginHorizontal": 16,
          "textAlign": "center",
        },
        Object {
          "color": "#E9122B",
          "marginTop": 24,
        },
      ],
    ]
  }>
  Bug!
</Text>
`;
